.travis.yml

```diff
@@ -4,56 +4,30 @@ sudo: false
 python:
   - "2.7"
   - "3.4"
+  - "3.5"
 
-# command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
+cache:
```

Comment:
It could be done but we'd need to query git to know.
Something equivalent to (in bash):

    [ "$(git log -n 1 --format=%h -- ete3/ncbi_taxonomy/ncbiquery.py)" == "$(git log -n 1 --format=%h)" ]